devops/openshift/queue-consumers-deploy.yml

```diff
@@ -212,21 +212,21 @@ objects:
                 failureThreshold: 3
                 successThreshold: 1
                 httpGet:
-                  path: /health/
+                  path: /health/timeout/1500
                   port: "${{PORT}}"
                 initialDelaySeconds: 10
                 periodSeconds: 10
-                timeoutSeconds: 1
+                timeoutSeconds: 3
```

Comment:
Just for my knowledge: why this `timeoutSeconds` has value 3 and the other one is 5?

Reply:
It was increased because the HTTP timeout was also increased.

_`timeoutSeconds`: The number of seconds of inactivity after which the probe times out and the container is assumed to have failed. The default is 1. This value must be lower than periodSeconds._
Source: https://docs.openshift.com/container-platform/4.9/applications/application-health.html